docs/10_using_dandi.md

```diff
@@ -57,12 +57,16 @@ or in a [Jupyter notebook on Dandihub](https://hub.dandiarchive.org).
 
 #### Using the Python CLI
 
-First install the Python client using `pip install dandi` in a Python 3.6+
+First install the Python client using `pip install dandi` in a Python 3.7+
 environment.
 
-1. Downloading a `Dandiset`.
-1. Downloading a subject.
-1. Downloading a file.
+
+1. Downloading a `Dandiset`, e.g.:
+`dandi download https://identifiers.org/DANDI:000004`
+1. Downloading data for a specific subject from a dandiset, e.g.:
+`dandi download https://api.dandiarchive.org/api/dandisets/000003/versions/draft/assets/?path=sub-YutaMouse20`
+1. Downloading a specific file from a dandiset (a link for the specific file could be found on the dandiarchive.org website), e.g.:
+`dandi download https://api.dandiarchive.org/api/dandisets/000004/versions/draft/assets/9d9f379d-9fd1-4872-8c49-3891dfb693ab/download/`
```

Comment:
for each of these it may be useful to describe the process by which a person finds this info (how do you get to know there is sub-YutaMouse20). consider a novice user, who has never uploaded or downloaded data from dandi.

it may be helpful to break this down into whether something is possible with the UI (search, view) or only the CLI (dandiset, subject) or both (asset).

Reply:
I can't find information about `sub-YutaMouse20` when running `dandi ls https://identifiers.org/DANDI:000004`. Do you have to go to dandiarchive.org website?

Reply:
Try with -r ?

Reply:
@yarikoptic - it works with `-r`, i didn't realize that that was for `03` and not for `04`